app/views/onboardings/preferences.html.erb

```diff
@@ -63,7 +63,7 @@
 
     <p class="text-secondary text-xs mb-4"><%= t(".preview") %></p>
 
-    <%= styled_form_with model: @user, data: { turbo: false } do |form| %>
+    <%= styled_form_with model: @user, method: :patch, data: { turbo: false } do |form| %>
```

Comment:
If I'm not mistaken, Rails should already be submitting a PUT request since the `@user` exists already and is not a new record. I'm not actually able to reproduce the original error either, so I'm wondering if there is an unrelated bug going on instead?

Reply:
Gotcha, I'm going to close out this PR and the corresponding issue for now as I don't think we've got a bug, but happy to reopen in the future if we can reproduce it more reliably!